cmd/shortener/repositories/repositories.go

```diff
@@ -0,0 +1,25 @@
+package repositories
+
+type LinkRepo interface {
+	GetLink(string) (string, bool)
+	CreateLink(string, string)
+}
+
+type MemoryLink struct {
+	links map[string]string
+}
+
+func NewMemoryLink() *MemoryLink {
```

Comment:
Чтобы реализовывать пакет через interface, следует его и передавать.
```suggestion
func NewMemoryLink() LinkRepo {
```
а структуру MemoryLink  лучше сделать приватной, чтобы обращаться можно было только через методы